docs/deploy/images.md

```diff
@@ -0,0 +1,21 @@
+# Container Images
+
+Images for the various KubeRay components are published at the following locations:
+
+1. [Quay.io](https://quay.io/organization/kuberay)
+2. [Dockerhub](https://hub.docker.com/u/kuberay)
+
+We recommend using Quay as the primary source for images as there are are no image-pull restrictions on Quay.io.
+
+## Stable versions
+
+For stable releases, use version tags (e.g. `kuberay/operator:v0.6.0`).
```

Comment:
Will `kuberay/operator:v0.6.0` pull images from Quay by default or should we change to `quay.io/kuberay/operator:v0.6.0`?

Reply:
We will need to update that -- the reason I didn't mention it was that the image isn't available on Quay yet. If you can push the images up there, I can update the docs accordingly!

Reply:
Done. I uploaded both operator and apiserver images to Quay manually.

Reply:
Updated